skins/common/spec/wikibits.spec.js

```diff
@@ -0,0 +1,217 @@
+/*global describe, it, expect*/
+describe('wikibits', function() {
+	'use strict';
+	mw.config = new mw.Map();
+	var url1 = '/index.php?title=MediaWiki:UserTags/code.js&action=raw&ctype=text/javascript';
+	var url1expect = '/index.php?title=MediaWiki:UserTags.js&action=raw&ctype=text/javascript';
+	var url1expect2 = '//dev.wikia.com/index.php?title=MediaWiki:UserTags.js&action=raw&ctype=text/javascript';
+	var url2 = '/index.php?title=User:TK-999/common.js&action=raw&ctype=text/javascript';
+	var url3 = 'http://dev.wikia.com/index.php?title=MediaWiki:UserTags/code.js&action=raw&ctype=text/javascript';
```

Comment:
dev.wikia.com is moved to dev.fandom.com, also fandom.com works on https
```suggestion
	var url3 = 'https://dev.fandom.com/index.php?title=MediaWiki:UserTags/code.js&action=raw&ctype=text/javascript';
```